ecsmanage/management/commands/ecsmanage.py

```diff
@@ -23,7 +23,7 @@ def add_arguments(self, parser):
         parser.add_argument(
             "cmd",
             type=str,
-            nargs="+",
+            nargs="...",
```

Comment:
It's worth surfacing that the [docs](https://docs.python.org/2/library/argparse.html#nargs) reference `argparse.REMAINDER` in their examples, but I wanted to avoid importing an extra module here.

Is this OK, or would it be best practice to reference `argparse.REMAINDER` directly?

Reply:
It would be tiny bit safer to use the `argparse.REMAINDER` constant rather than the string literal. I highly doubt that the value of `argparse.REMAINDER` would change, but as a general rule of thumb "magic" string literals should be avoided.

The `parser` that is being passed to us is an `argparse.ArgumentParser` so I don't see any issue with importing the module that we are already using code from implicitly.